usaspending_api/core/validator/award_filter.py

```diff
@@ -14,7 +15,8 @@
     {'name': 'program_numbers', 'type': 'array', 'array_type': 'text', 'text_type': 'search'},
     {'name': 'psc_codes', 'type': 'array', 'array_type': 'text', 'text_type': 'search'},
     {'name': 'recipient_scope', 'type': 'enum', 'enum_values': ('domestic', 'foreign')},
-    {'name': 'recipient_search_text', 'type': 'array', 'array_type': 'text', 'text_type': 'search', 'max': 1, 'min': 1},
+    {'name': 'recipient_search_text', 'type': 'array', 'array_type': 'text',
+     'text_type': 'search', 'array_max': 1, 'array_min': 1},
```

Comment:
(Adding discussed comments to document and track)
Originally, the `min` and `max` were "top-level" attributes while the type-specific min and max (`array_max`, `object_min`, ...) were intended for the sub-level. If that can be maintained it will probably allow more consistency across all models and different places in the code.

However, if it requires a non-trival amount of effort to revert back then keep as-is for now and we can reevaluate in the future.

Reply:
Also, for the sake of getting this ticket out, I'd like to push this particular change towards a later, phase 2 assessment of tinyshield once we've had it in the wild a little bit.